README.rst

```diff
@@ -1,3 +1,13 @@
+==============
+mongo-connector-solr
+==============
+This project is based on original mongo-connector::https://github.com/mongodb-labs/mongo-connector
+
+Change log:
+-----------
+Add support to DBRef whose datatype is "bson.dbref.DBRef".
+
+
```

Comment:
Remove this, the change-log is https://github.com/mongodb-labs/mongo-connector/blob/master/CHANGELOG.rst.